### ISSUE TITLE
QA: Sending Transactions: App drains battery

### ISSUE THREAD

**Issue by Tas4tari** [tari-project/wallet-android#247]
When I started sending and receiving transactions, it became really power hungry and drains the battery fast.

Device Details: Android 9

**Comment by talmahaj** [comment]
@kukabi this problem is back apparently, did you see that screenshot in Telegram? :(

**Comment by kukabi** [comment]
@talmahaj Yeah i saw the screenshot, I'll have a look at what could be causing it in the app & ping the FFI team.

**Comment by Tas4tari** [comment]
Testing on Huawei P30 lite, app version: 0.1.14
Will leave the background processing without my device being on charge and will monitor battery usage through the afternoon.

![Screenshot_20200601_153129_com huawei systemmanager](https://user-images.githubusercontent.com/50987039/83414590-12d2cb00-a41e-11ea-8e2b-27c8fc6e1828.jpg)

**Comment by Tas4tari** [comment]
Last night I received the following message:
![Screenshot_20200601_234921_com huawei android launcher](https://user-images.githubusercontent.com/50987039/83485953-6e499b00-a4a8-11ea-86fe-01c55071e031.jpg)

![Screenshot_20200601_220427_com huawei android launcher](https://user-images.githubusercontent.com/50987039/83486226-10698300-a4a9-11ea-87fa-d4cec1e4fe9f.jpg)


![Screenshot_20200601_234943_com huawei systemmanager](https://user-images.githubusercontent.com/50987039/83485974-76093f80-a4a8-11ea-9a2e-8866ba2e9c89.jpg)

**Comment by Tas4tari** [comment]
I left the app running in the background over night and when I woke up this morning the battery hadn't died but it looks like the background process had stopped on its own: cc @kukabi 
![Screenshot_20200602_063605_com huawei systemmanager](https://user-images.githubusercontent.com/50987039/83486066-b49efa00-a4a8-11ea-813e-222059adcf55.jpg)

**Comment by kukabi** [comment]
Thanks a lot @Tas4tari, I think we should find a way of reducing activity in background mode. I'll consult the FFI team.

**Comment by Anziskavds** [comment]
Tested successfully on Andriod 10 on a Samsung S9 version 0.1.15. I have been testing since this morning and send and receive transactions and my battery life is still fine.

![Screenshot_20200622-144602_Device care.jpg](https://images.zenhubusercontent.com/5cdd14f21f980125322a448a/ea94fba7-8587-4983-a20a-0cda284270a4)

**Comment by Tas4tari** [comment]
@kukabi Re-opening this issue as I had received an error for power intensive app found about 2 - 2,5 hours after opening the app. At that point the app was running in the background.

**App version:**
0.2.0-jnLibs 0.13.0

**Logs:**
[Uploading ffi_logs_FC2EEE595F38DE76461E0E17197075E42FE12A9B02E283EB9B8AB5057C2F9B0F.zip…]()


**Screenshots:**
![image-b1b8e3f5-f32d-4ef9-ba2c-32c0f52d64ac](https://user-images.githubusercontent.com/50987039/85419122-f7b53000-b571-11ea-92ee-a1c5e6b7bb41.jpg)
![image-eb7dc8ac-62f3-4b82-b0f6-ffeef230f86b](https://user-images.githubusercontent.com/50987039/85419186-0ac80000-b572-11ea-8859-e57ecd12a107.jpg)
![image-38801ce9-0f1d-4dfa-b9be-34b9bfa6b32f](https://user-images.githubusercontent.com/50987039/85419206-13203b00-b572-11ea-9f33-742226d7ec86.jpg)

**Comment by Tas4tari** [comment]
Changing the release of this as it was re-opened just before deployment. General consensus on Telegram is that there has been improvement for some users, whilst others still report high battery usage

**Comment by Tas4tari** [comment]
**02 August:**
**User:**
C8H10N4O2
**Device:**
Samsung S8
**Comments:**
Android app absolutely demolished my battery
Kutsal, [Aug 2, 2020 at 7:06:31 AM]:
hi, what's your device?

Michael Anthony, [Aug 2, 2020 at 8:29:06 AM]:
Yes I had same problem on Samsung s6 model ,

C8H10N4O2, [Aug 2, 2020 at 8:52:30 AM]:
S8

cc @kukabi

**Comment by Tas4tari** [comment]
@Tas4tari To discuss with the Protocol team

**Comment by Tas4tari** [comment]
I have spoken to @philipr-za with regards to the battery draining issue, he did indicate there could be some potential tuning and fixes to the comms layer that could improve this, however this will require some effort with regards to determining what is eating power within the wallet app. 
They are currently focused on working on the network stability and a few other tasks that will take priority over this.

I will move this ticket to the Tari repo in the interim

cc @kukabi @talmahaj
